orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// needs to be included first to get around onnxruntime\cmake\external\onnx\onnx/common/constants.h(14): error C2513: 'bool': no variable declared before '='
```

Comment:
I guess it is not needed anymore

Reply:
Thanks, I saw your early PR that changing the OPTIONAL constant. 
I'll revert it then.